src/connection.js

```diff
@@ -708,10 +729,28 @@ class Connection extends EventEmitter {
     });
   }
 
+  // Returns false to apply backpressure.
   sendDataToTokenStreamParser(data) {
     return this.tokenStreamParser.addBuffer(data);
   }
 
+  pauseRequest(request) {
```

Comment:
This is only an internal method that is called from Request.pause(). It has to check whether this Request object represents the currently active request. I will add a comment.